src/graphql/queries.js

```diff
@@ -731,17 +733,31 @@ const refreshLoggedInUser = async (data) => {
   }
 };
 
+const maybeRefreshAccessToken = async (currentToken) => {
+  const { exp } = JSON.parse(atob(currentToken.split('.')[1]));
+  const shouldUpdate = moment(exp * 1000).subtract(1, 'month').isBefore(new Date);
+  if (shouldUpdate) {
+    const { token } = await api.refreshToken(currentToken);
+    window.localStorage.getItem('accessToken', token);
```

Comment:
@clarete maybe your intent was to do setItem there :-)

Reply:
@znarf thank you so much for the catch! I'll fix this today! <o/